README.md

```diff
@@ -1,30 +1,41 @@
-
 # Customs Financials Frontend
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Coverage](https://img.shields.io/badge/test_coverage-90%-green.svg)](/target/scala-2.11/scoverage-report/index.html) [![Accessibility](https://img.shields.io/badge/WCAG2.2-AA-purple.svg)](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag)
 
-A frontend component for the CDS Financials project which aims to provide financial services for customs 
-transactions.
+A micro-frontend service - This service provides a hub/entry point to access the different financial services for HMRC customs.
 
-| Path                                                                   | Description                                                                                       |
-| ---------------------------------------------------------------------  | ------------------------------------------------------------------------------------------------- |
-| GET  /import-vat                                                       | Retrieve all import vat certificates                                                           |                
-| GET  /postponed-vat                                                    | Retrieve all postponed vat statements                                                          |                
-| GET  /adjustments                                                      | Retrieve all securities statements                                                           |                
+This service is built following GDS standards to [WCAG 2.2 AA](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag)
 
+We use the [GOV.UK design system](https://design-system.service.gov.uk/) to ensure consistency and compliance through the project
 
 This applications lives in the "public" zone. It integrates with:
 
-* Secure Payments Service (SPS) / Enterprise Tax Management Platform (ETMP) via the [Customs Financials API](https://github.com/hmrc/customs-financials-api)
-* Secure Document Exchange Service (SDES) bulk data API via the [SDES proxy](https://github.com/hmrc/secure-data-exchange-proxy)
+Secure Payments Service (SPS) / Enterprise Tax Management Platform (ETMP) via the [Customs Financials API](https://github.com/hmrc/customs-financials-api)
+Secure Document Exchange Service (SDES) bulk data API via the [SDES proxy](https://github.com/hmrc/secure-data-exchange-proxy)
```

Comment:
Could you please make sure that this should display as a next line (as these are two different things) in Preview mode cause right now line 13 and 14 are making a one single line?